src/OverpassFrontend.js

```diff
@@ -249,6 +249,13 @@ class OverpassFrontend {
       this.errorCount = 0
     }
 
+    let osm3sMeta = {
+      osm3s_version: results.version,
+      generator: results.generator,
+      osm_timestamp: results.osm3s.timestamp_osm_base,
```

Comment:
here there might as well be some timestamp for "area" extension available if you include the "area" as BBOX... don't have a sample handy but could produce one

Reply:
but i like these changes. they would surely make a better solution given the subrequest stuff

Reply:
As an example for generator-side extension metadata you might want to try this query in overpass-turbo:

```
[out=json];
( area[name="Wien"]; )->.searchArea;
  node
    [amenity="drinking_water"]
    (area.searchArea);
  out meta;
```

it utilizes the "area" extension of osm3s and results in this

```
{
  "version": 0.6,
  "generator": "Overpass API 0.7.55.4 3079d8ea",
  "osm3s": {
    "timestamp_osm_base": "2018-08-31T20:32:02Z",
    "timestamp_areas_base": "2018-08-31T19:27:02Z",
    "copyright": "The data included in this document is from www.openstreetmap.org. The data is made available under ODbL."
  },
  "elements": [
  ...
```

note the `timestamp_areas_base` key of `osm3s` whereas a "simple" bounding box query would not produce that key as the extension is not involved.

I don't know how many and which extensions are available - and overpass-frontend should not assume the existence of one or any of them beyond maybe osm_base as these are enabled by the resulting query itself.
Thus I would prefer some transparent implementation.